.github/l10n/check_l10n_issues.py

```diff
@@ -21,7 +21,7 @@
     exceptions = json.load(f)
 
 # Load the .ts file
-ts_file = os.path.join(vpn_root_folder, "translations", "mozillavpn_en.ts")
+ts_file = os.path.join(vpn_root_folder, "translations", "en", "mozillavpn_en.ts")
```

Comment:
Is it expected that the linter is now failing? Would it fix itself once merged?

`Updating 'translations/mozillavpn_en.ts'`, which means the file in `en` doesn't exist.

Reply:
I think this line should be reverted. This script extracts its own .ts file in `translations/mozillavpn_en.ts`. As far as I understand, the import script relies on `translations.pri` for extracting them in subfolders, and that's not needed here.

Reply:
Right. This file now is into an "en" folder.